identity/src/main/java/com/android/identity/DeviceResponseParser.java

```diff
@@ -224,13 +220,14 @@ PublicKey parseIssuerSigned(
                     Util.coseSign1GetData(issuerAuthDataItem));
             DataItem mobileSecurityObject = Util.cborExtractTaggedAndEncodedCbor(
                     mobileSecurityObjectBytes);
+            byte[] encodedMobileSecurityObject = Util.cborEncode(mobileSecurityObject);
```

Comment:
I think the calls to `Util.cborExtractTaggedAndEncodedCbor()` and `Util.cborEncode()` decodes and then re-encodes the bstr. I think you can remove both calls and just replace it with
```
 byte[] encodedMobileSecurityObject = Util. cborExtractTaggedCbor(mobileSecurityObjectBytes);
```

Reply:
Thanks for the clarification!